using_images/s2i_images/ruby.adoc

```diff
@@ -33,32 +33,47 @@ link:https://github.com/openshift/sti-ruby/tree/master/2.3[2.3] of Ruby.
 [[s2i-images-ruby-images]]
 == Images
 
+ifdef::openshift-online[]
+RHEL 7 images are available through Red Hat's subscription registry:
```

Comment:
s/Red Hat's subscription registry/the Red Hat Registry/

See https://access.redhat.com/containers/#/faq.